src/main/resources/assets/computercraft/lua/rom/motd.txt

```diff
@@ -28,7 +28,17 @@ Use "monitor" to run a program on a attached monitor.
 View all attached peripherals with "peripherals".
 Use "time" to see the in-game time.
 You can set the label of a computer with "label set".
-A computer needs a label to keep its files if it got destroyed.
+A computer needs a label to keep its files if it gets destroyed.
 You can disable auto completion in the shell with "set shell.autocomplete false".
 You can disable auto completion in edit with "set edit.autocomplete false".
 Feeling creative? Use a printer to print a book!
+Files beginning with a "." character are hidden from "list" by default.
+Running "set" lists all currently set settings.
```

Comment:
"all available settings" maybe? It's a little unwieldy a sentence otherwise.

Reply:
It's not necessarily all available settings, as some user programs may use the `settings` API too, and won't be listed in that unless you explicitly set them.